metadata-db-app/src/cmr/metadata_db/migrations/089_add_granule_delete_index.clj

```diff
@@ -0,0 +1,22 @@
+(ns cmr.metadata-db.migrations.089-add-granule-delete-index
+  (:require
+   [config.mdb-migrate-helper :as mh]))
+
+(defn up
+  "Migrates the database up to version 89."
+  []
+  (println "cmr.metadata-db.migrations.089-add-granule-delete-index up...")
+  (doseq [table-name (mh/get-concept-tablenames :granule)]
+    (try
+      (mh/sql (format "create index %s_DD ON %s (deleted, delete_time)" table-name table-name))
```

Comment:
don't you need a different index for small_providers, one that includes the provider_id?

Reply:
Yes, Thank you!  I have fixed the mistake.